core/app/models/spree/variant/pricing_options.rb

```diff
@@ -19,12 +22,38 @@ def initialize(desired_attributes = {})
         @desired_attributes = self.class.default_price_attributes.merge(desired_attributes)
       end
 
+      def arel_conditions
```

Comment:
I'm considering naming it something else as it's anything that can be used as a `XXXX` to `variant.prices.where(XXXX)`. For the case we're looking at here, it could even be a hash.